ansible/roles/os_zabbix/vars/template_openshift_master.yml

```diff
@@ -2,6 +2,18 @@
 g_template_openshift_master:
   name: Template Openshift Master
   zitems:
+  - key: origin-master.version
+    description: Show origin-master rpm version
+    value_type: string
+    applications:
+    - Openshift Cluster
+
+  - key: atomic-openshift-master.version
```

Comment:
Hmm, this naming scheme of the key doesn't really fit in with the rest of the master keys.

I didn't notice it in the other PR (sorry).

What are your thoughts on changing the key to be something like `openshift.master.version.atomic-openshift` ?

And the one above would be something like `openshift.master.version.origin-master`.

Obviously we'd want to change the node keys as well. The `docker.version` is prefect as is, though.

The reason that keeping the scheme the same is that we display metrics in zabbix using the "latest data" and its search is quite limited.

So if this is a different pattern, it won't show up if we search for "openshift.master" metrics.